### PR TITLE
Added modern node versions to travis ci

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run test
+  - npm run lint
```
Comment (alexfernandez) [inline .travis.yml]: These two scripts are not configured and are failing on Travis.

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,32 @@
+sudo: false
+
 language: node_js
 
-# test on two node.js versions: 0.8 0.10
 node_js:
-  - 0.8
-  - 0.10
```
Comment (alexfernandez) [inline .travis.yml]: Why remove 0.10 and not add 0.12? They are still in use in many places.

**Comment by amilajack** [comment]
Thanks for letting me know. Just added them!

**Comment by amilajack** [comment]
@alexfernandez so sorry about that! Didn't see one of your comments. I removed the tests and now everything should pass

**Comment by alexfernandez** [comment]
+1, merging.

**Pull request merged**